libflux/flux-core/src/semantic/flatbuffers/types.rs

```diff
@@ -680,6 +709,10 @@ mod tests {
         test_serde("[A]");
     }
     #[test]
+    fn serde_vector_type() {
+        test_serde("[A]");
```

Comment:
This test uses the type parser to create a monotype and roundtrip it. You would need to add support to the parser for vector types to use it. We should talk about whether you should do this, or just test flatbuffers serialization in an ad hoc way.

Reply:
Sure, let's discuss this when you have some time today. Thanks